README.md

```diff
@@ -256,14 +256,45 @@ If the standard is still a draft, it belongs here.
 Use a name beginning with `dom-` and include a link to the standard as the "Project" link in the header.
 When it graduates draft mode, we may remove it from DefinitelyTyped and deprecate the associated `@types` package.
 
-#### I want to update a package to a new major version
+#### How do DefinitelyTyped package versions relate to versions of the corresponding library? 
 
-If you intend to continue updating the older version of the package, you may create a new subfolder with the current version e.g. `v2`, and copy existing files to it. If so, you will need to:
+_NOTE: The discussion in this section assumes familiarity with [Semantic versioning](https://semver.org/)_
+
+Each DefinitelyTyped package is versioned when published to NPM. The [automated tools](https://github.com/Microsoft/types-publisher) that publish typings packages to NPM will set the typings package's version using the version number listed in the first line of the typings file. For example, below are the first few lines of the latest (as of late 2018) [node.js typings file](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/node/index.d.ts) for node.js library versions `10.12.x`.  
+
+```javascript
+// Type definitions for Node.js 10.12
+// Project: http://nodejs.org/
+// Definitions by: Microsoft TypeScript <https://github.com/Microsoft>
+//                 DefinitelyTyped <https://github.com/DefinitelyTyped>
+//                 Alberto Schiabel <https://github.com/jkomyno>
+```
+
+Because `10.12` is at the end the first line, the NPM version of the `@types/node` package will also be `10.12.x`. Note that the first-line comment in the typings file should only contaiin major/minor versions (e.g. `10.12`) and should not contain a patch version (e.g. `10.12.4`). This is because only the major and minor release numbers are aligned between library packages and typings packages. The patch release number of the typings package (e.g. `.0` in `10.12.0`) is initialized to zero by DefinitelyTyped and is incremented each time a new `@types/node` package is published to NPM for the same major/minor version of the corresponding library.
```

Comment:
Haven't read the whole thing yet, but a few requests:

- One sentence per line
- contaiins -> contains
- typings files -> type declarations

Reply:
Thanks @DanielRosenwasser for feedback! I updated in response: 
* Fixed typos: contaiin, udpated, udpate, mainatiners
* One sentence per line, except bullet points (newlines in markdown bullet points will show to users). Note that much of the existing content didn't adhere to this convention, but I didn't edit that existing content-- assume this is OK. 
* Removed "typings", replaced with either "type definition(s)" or "type definition package" depending on context

Happy to make more edits, just let me know.

Reply:
I'm not sure why this PR is stuck, but perhaps it's because this conversation isn't resolved, so will resolve it now. Feel free to re-open review if there are more changes you'd like to see.